book/src/slasher.md

```diff
@@ -71,7 +71,7 @@ If an MDBX database is already found on disk, then Lighthouse will try to use it
 in a log at start-up:
 
 ```
-INFO Slasher backend overriden    reason: database exists, configured_backend: lmdb, overriden_backend: mdbx
+INFO Slasher backend overridden    reason: database exists, configured_backend: lmdb, overridden_backend: mdbx
```

Comment:
I agree that this is a typo but the typo exists in the code rather than the documentation: this logging output arises here:

https://github.com/sigp/lighthouse/blob/441fc1691b69f9edc4bbdc6665f3efab16265c9b/beacon_node/src/lib.rs#L95